plugins/services/src/js/reducers/serviceForm/FormReducers/__tests__/Artifacts-test.js

```diff
@@ -7,9 +7,9 @@ describe("Artifacts", function() {
   describe("#FromReducer", function() {
     it("emits correct JSON", function() {
       const batch = new Batch([
-        new Transaction(["fetch"], 0, ADD_ITEM),
+        new Transaction(["fetch"], { uri: "http://mesosphere.io" }, ADD_ITEM),
         new Transaction(["fetch", 0, "uri"], "http://mesosphere.io", SET),
-        new Transaction(["fetch"], 1, ADD_ITEM),
+        new Transaction(["fetch"], { uri: "http://mesosphere.com" }, ADD_ITEM),
```

Comment:
This is just a test – no need for https, but we might want to use `example.com` to avoid confusion.